examples/opensora_hpcai/opensora/pipelines/train_pipeline.py

```diff
@@ -352,7 +352,8 @@ def compute_loss(
 
         # Learn the variance using the variational bound, but don't let it affect our mean prediction.
         patch_mask = temporal_mask[:, :, None, None] * spatial_mask[:, None, :, None]
-        patch_mask = self.unpatchify(ops.tile(patch_mask, (1, 1, 1, D)))  # b c t h w
+        pm_dtype = patch_mask.dtype
+        patch_mask = self.unpatchify(ops.tile(patch_mask.to(ms.int32), (1, 1, 1, D)).to(pm_dtype))  # b c t h w
```

Comment:
as a temporary fix due to MS/CANN bug, maybe not merge to master?

Reply:
the performance of ops.tile / repeat in uint8 is no better than that in int32, so it's worthy.

Reply:
though the time for uint8/int32 is same, casting to int32 and casting back will add two extra casting operations, may not be nesessary